### PR TITLE
Adapted the vsso2ttl tooling to vsso-core ontology

### PR DESCRIPTION
**Pull request by danielwilms**
As proposed in the w3c/vsso repo, there are two namespaces
to be used. one for vsso-core and one for concepts generated
out of VSS. This commits reflects that by:

* referring to the core concepts defined in w3c/vsso
* defining the ontology
* removing unused code

### CODE REVIEW

File: contrib/vspec2ttl/vspec2ttl.py [hunk 1]
```diff
@@ -55,109 +80,45 @@ def usage():
...
+
+
+        vsso: rdf:type owl:Ontology ;
+            dcterms:title "VSSo: Vehicle Signal Specification Ontology";
+            vann:preferredNamespaceUri "https://github.com/w3c/vsso#" ;
+            dcterms:description "This ontology describes the car's attributes, branches and signals defined in the Vehicle Signal Specification." ;
+            dcterms:license <http://creativecommons.org/licenses/by/4.0/> ;
+            dcterms:creator "Benjamin Klotz"^^xsd:string ;
+            dcterms:creator "Raphael Troncy"^^xsd:string ;
+            dcterms:creator "Daniel Wilms";
```
Comment (erikbosch) [inline contrib/vspec2ttl/vspec2ttl.py:102]: No xsd:string needed for Daniel?

**Comment by erikbosch** [comment]
I saw some warnings in the build log at https://github.com/COVESA/vss-tools/runs/5025002024?check_suite_focus=true

`AttributeError: 'VSSNode' object has no attribute 'enum'`

**Comment by danielwilms** [comment]
create maintenance branch to support older version and fix this one for 3.0

**Review by erikbosch — APPROVED** [review]
Looks good from my perspective

**Comment by danielwilms** [comment]
approved in meeting on 2022-03-01

@erikbosch, could you merge please?

**Pull request merged**